resources/public/api/conf/2.0/uk_property_non_fhl_period_summary_create.raml

```diff
@@ -0,0 +1,46 @@
+is:
+  - headers.acceptJson
+  - headers.contentTypeJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.ruleTaxYearNotSupported
```

Comment:
The error scenario description needs to be slightly different because there is not only a minimum but a maximum tax year and the tax year is inferred (by the backend) from the period dates:
```suggestion
  - errors.rulePeriodTaxYearNotSupported
```